### PR TITLE
Remove Pseudo-Huber loss, it duplicates SoftLOne

### PR DESCRIPTION
**Pull request by efernandez**
I just realized the **Pseudo-Huber** loss is the same as the **SoftLOne** loss, already provided by Ceres.

In this PR I remove the **Pseudo-Huber** loss and add a comment in the **SoftLOneLoss** plugin that mentions it's the same as the **Pseudo-Huber** loss from https://en.wikipedia.org/wiki/Huber_loss#Pseudo-Huber_loss_function

### CODE REVIEW

**Comment by efernandez** [comment]
Sorry for the back and forth with this.

Pinging @svwilliams @ayrton04 as reviewers, since I can't add you as such.

**Review by svwilliams — APPROVED** [review]

**Review by ayrton04 — APPROVED** [review]

File: fuse_loss/include/fuse_loss/loss_function.h [hunk 1]
```diff
@@ -227,38 +227,10 @@ class GemanMcClureLoss : public ceres::LossFunction
...
-private:
-  const double b_;
-  const double c_;
-};
-
 // Welsch, similar to Tukey loss, it tries to reduce the effect of large errors, but it does not suppress outliers as
 // Tukey might do.
 //
-// The terms is computed as:
+// The term is computed as:
```
Comment (efernandez) [inline fuse_loss/include/fuse_loss/loss_function.h:233]: I just sent another commit to this PR to fix this minor typo.
